### DISCUSSION TITLE
How do I connect my Github Desktop to my Github Enterprise?

### DISCUSSION THREAD

**Discussion by Plavixal2023** [community/community#125730] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

<title>How do I connect my Github Desktop to my Github Enterprise?

**Comment by Abhishek22K** [comment]
To connect GitHub Desktop to your GitHub Enterprise instance, follow these steps:

Open GitHub Desktop:
Launch GitHub Desktop on your computer.

Sign in to GitHub Enterprise:

Go to the menu bar and select File > Options (on Windows) or GitHub Desktop > Preferences (on macOS).
In the Preferences window, go to the Accounts tab.
Click on the Sign in to GitHub.com button and then select Enterprise.
Enter GitHub Enterprise Server URL:

Enter the URL of your GitHub Enterprise server (e.g., https://github.yourcompany.com).
Click Continue.
Authenticate:

You will be redirected to your GitHub Enterprise server's authentication page.
Enter your username and password, or use the authentication method configured for your enterprise (e.g., SSO, OAuth, etc.).
Authorize GitHub Desktop to access your GitHub Enterprise account.
Finish Setup:

Once authenticated, GitHub Desktop will be connected to your GitHub Enterprise account.
You can now clone repositories from your GitHub Enterprise instance, create new repositories, and push/pull changes.
Troubleshooting Tips
Network Issues: Ensure that your computer can reach your GitHub Enterprise server over the network.
SSL Certificates: If your GitHub Enterprise server uses a self-signed SSL certificate, you might need to install the certificate on your computer or configure GitHub Desktop to trust the certificate.
Firewall/Proxy Settings: Check if there are any firewall or proxy settings that might block access to your GitHub Enterprise server.
